sdk/cosmosdb/cosmos/src/queryIterator.ts

```diff
@@ -118,6 +123,18 @@ export class QueryIterator<T> {
         }
       }
 
+      if (
+        this.container &&
+        this.clientContext.enableEncryption &&
+        this.resourceType === ResourceType.item &&
+        response.result &&
+        response.result.length
+      ) {
+        for (let item of response.result) {
+          item = await this.container.encryptionProcessor.decrypt(item, diagnosticNode);
```

Comment:
instead of using container.encryptionProcessor , encryptionProcessor can be directly used